main.cpp

```diff
@@ -1,475 +1,581 @@
+    //train staff
+    void staffTraining(Employee employee, std::string trainingRequired);
+};
+
 /*
-Part 1d assignment:
-1) Fill in #5 - #9 below with plain-english UDTs for the 5 
-properties you created for UDT #10. 
-    example: 
-        If #10's first property was 'Engine', then `Thing 5)` 
-        will be `Engine`. 
-        You will need to provide 5 properties and 3 member 
-        functions of that Engine object in plain English.
-        Remember to pick properties that can be represented 
-        with 'int float double bool char std::string'.
-
-2) write the name of the primitive type you'll be using after 
-each property for UDTs 5 - 9.
-    You already did this for UDTs 1-4 in Part 1c.
-    Pick properties that can be represented with 
-    'int float double bool char std::string'
-    example: 
-        Display:
-            Number of Pixels (int)
-            Amount of Power consumed (milliwatt-hours) (float)
-            Brightness (double)
-            width in cm (int)
-            height in cm (int)
+Thing 2) School
+5 properties:
+    1) number of classrooms (int)
+    2) position in leauge table (int)
+    3) number of teachers (int)
+    4) number of pupils (int)
+    5) student average grade (float)
+3 things it can do:
+    1) train teachers
+    2) students christmas play
+    3) close for holidays
 */
 
+struct School
+{
+    //number of classrooms (int)
+    int numClassrooms = 111;
+    //position in leauge table (int)
+    int leagueTablePosition = 13;
+    ///number of teachers (int)
+    int numTeachers = 54;
+    //number of pupils (int)
+    int numPupils = 1111;
+    //student average grade (float)
+    float aveGrade = 3.45f;
+
+    //train teachers
+    void teacherTraining(bool internalTraining, std::string trainingRequired);
+    //students christmas play
+    void christmasPlay(int numberOfStudentsInCast, std::string nameOfPlay, int rehearsalDays);
+    //close for holidays
+    void holidayClosure(int howManyDaysClosed, std::string nameOfHoliday);
+};
+
 /*
-Thing 5)
+Thing 3) Farm
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) number of sheep (int)
+    2) number of cows (int)
+    3) crop yeild (double)
+    4) number of farmers (int)
+    5) area in hectare (double)
 3 things it can do:
-    1)
-    2)
-    3)
- */
+    1) sell sheep
+    2) harvest crop
+    3) construct new barn
+*/
+
+struct Farm
+{
+    //number of sheep (int)
+    int numSheep = 1234;
+    //number of cows (int)
+    int numCows = 321;
+    //crop yeild (double)
+    double cropYield = 76543.21;
+    //number of farmers (int)
+    int numFarmers = 6;
+    //area in hectare (double)
+    double farmLandAreaInHectare = 84.3;
+
+    //sell sheep
+    float getTurnoverSheepSale(float priceOfSheep, int numSheep); //returns the income from the sale of this farms sheep
```

Comment:
Your function names do not match the plain-english descriptions.
They should match almost 1:1
- [ ] revise